app/integration/runtime/src/test/java/io/syndesis/integration/runtime/handlers/ConnectorStepHandlerPollEnrichIT.java

```diff
@@ -174,6 +175,7 @@ public void shouldSupportPollEnriching() throws Exception {
                     .putConfiguredProperty("period", "1000")
                     .build(),
                 new Step.Builder()
+                    .id("expected")
```

Comment:
why 'expected' ?

Reply:
To [assert it easier](https://github.com/syndesisio/syndesis/pull/9385/files#diff-fda11763c721dc439cde8d9d6ff6e8f294d8f4b75e24f7e9975e100f9efb32f4R210), otherwise it gets a randomly generated id, so we can't assert that it's this step that we keep track of. The test now covers a case where captured messages map was missing output from poll enrich.